assets/javascripts/kitten/components/buttons/button-tag.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import { ButtonTag } from 'kitten/components/buttons/button-tag'
+
+
+describe('ButtonTag with default props', () => {
+  const defaultComponent = shallow(
+    <ButtonTag />)
+
+  it('renders <div class="k-ButtonTag">', () => {
+    expect(defaultComponent.find('.k-ButtonTag')).to.have.length(1)
+  })
+
+  it('has a default tag', () => {
+    expect(defaultComponent).to.have.tagName('button')
+  })
+
+  it('has an is-selected class', () => {
```

Comment:
L'intitulé du test n'est pas en adéquation avec le test en lui-même. Tu annonces qu'il devrait y avoir la classe `is-selected` mais tu testes qu'il n'y en a pas (`length(0)`).